### ISSUE TITLE
count inconsistency?

### ISSUE THREAD

**Issue by kikocorreoso** [jdevera/pylabeador#6]
When you do the following:

```python
from pylabeador import syllabify_with_details

word = syllabify_with_details("melón")
print(word.accented)
print(word.stressed)
```
The `accented` 'counter' starts from 0 and the `stressed` 'counter' starts from 1.

From a user point of view maybe it has more sense that in both cases the counter starts from one.

If you agree I can prepare a PR fixing this.

**Comment by jdevera** [comment]
Thank you for checking out the tool and taking the time to file this issue!

It seems like a good idea to report both `stressed` and `accented` as 1-based counts. I'll gladly take you PR :)

**Comment by jdevera** [comment]
Having second thoughts here, considering that the accented element is the position of the character in the word that is accented, and to me that makes a lot of sense to be 0-based, since you can use to access the accented letter directly.

Perhaps the same logic should apply to the syllables. If you think about it, the users of this are going to be programmers, since whoever uses this as a command-line tool will not even get that information anyway.

So at the risk of sounding bi-polar here, I'm leaning now more towards the 0-based indices for both.

**Comment by kikocorreoso** [comment]
:smile: 

Bipolarity is on fashion these days.

I'm +0 to 0-based (for programmers) or 1-based (for humans, *como díría aquel*). Just decide what do you prefer.

But I'm + :100: to be consistent here, or 0-based or 1-based, but not both.

**Comment by jdevera** [comment]
Okay let's do 0-based.

**Comment by kikocorreoso** [comment]
Closed by #9
